lib/worker.js

```diff
@@ -0,0 +1,11 @@
+var newJasmine = require("./new"),
+    runJasmine = require('./run'),
+    noop = require("lodash.noop");
+
+module.exports = exports = function() {
+  process.on('message', function(env) {
+    var jasmine = newJasmine();
+    jasmine.onComplete(noop);
```

Comment:
This should probably just be an inlined empty function instead of bringing in a new dependency just for an empty function.

Reply:
![image](https://user-images.githubusercontent.com/5212215/57058681-081b9480-6c66-11e9-9911-85fc32ef0684.png)